Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity_test.py

```diff
@@ -0,0 +1,1004 @@
+    assert res.outputs == ACTIVITIES_BY_ID_DATA_CONTEXT
+
+
+def test_list_files_command(requests_mock):
+    from MicrosoftCloudAppSecurity import list_files_command
+    requests_mock.get('https://demistodev.eu2.portal.cloudappsecurity.com/api/v1/files/5f077ebfc3b664209dae1f6b',
+                      json=FILES_BY_ID_DATA)
+    res = list_files_command(client_mocker, {'file_id': '5f077ebfc3b664209dae1f6b'})
+    assert res.outputs == FILES_BY_ID_DATA_CONTEXT
+
+
+def test_list_users_accounts_command(requests_mock):
+    from MicrosoftCloudAppSecurity import list_users_accounts_command
+    requests_mock.get('https://demistodev.eu2.portal.cloudappsecurity.com/api/v1/entities/',
+                      json=ENTITIES_BY_USERNAME_DATA)
+    res = list_users_accounts_command(client_mocker,
+                                      {'username': '{ "id": "7e14f6a3-185d-49e3-85e8-40a33d90dc90",'
+                                                   ' "saas": 11161, "inst": 0 }'})
+    assert ENTITIES_BY_USERNAME_DATA_CONTEXT == res.outputs
+
+
+ALERT_BY_ID_DATA_CONTEXT = {
+    'URL': 'https://demistodev.portal.cloudappsecurity.com/#/alerts/5f06d71dba4289d0602ba5ac',
+    '_id': '5f06d71dba4289d0602ba5ac',
+    'account': [{'em': 'dev@demistodev.onmicrosoft.com',
+                 'entityType': 2,
+                'id': '2827c1e7-edb6-4529-b50d-25984e968637',
+                 'inst': 0,
+                 'label': 'demisto dev',
+                 'pa': 'dev@demistodev.onmicrosoft.com',
+                 'saas': 11161,
+                 'type': 'account'}],
+    'comment': 'null',
+    'contextId': 'ebac1a16-81bf-449b-8d43-5732c3c1d999',
+    'description': "File policy 'block png files' was matched by 'image (2).png'",
+    'file': [{'id': 'd10230e2-52db-4ec8-815b-c5484524d078|501f6179-e6f9-457c-9892-1590dee07ede',
+               'label': 'image (2).png',
+               'type': 'file'}],
+    'handledByUser': 'null',
+    'idValue': 15728642,
+    'isSystemAlert': False,
+    'is_open': False,
+    'policy': {'id': '5f01dce13de79160fbec4150',
+               'label': 'block png files',
+               'policyType': 'FILE',
+               'type': 'policyRule'},
+    'policyRule': [{'id': '5f01dce13de79160fbec4150',
+                    'label': 'block png files',
+                    'policyType': 'FILE',
+                    'type': 'policyRule'}],
+    'resolveTime': '2020-07-12T07:48:40.975Z',
+    'service': [{'id': 15600,
+                'label': 'Microsoft OneDrive for Business',
+                 'type': 'service'}],
+    'severityValue': 0,
+    'statusValue': 1,
+    'stories': [0],
+    'threatScore': 19,
+    'timestamp': 1594283802753,
+    'title': 'block png files',
+    'user': [{'id': 'dev@demistodev.onmicrosoft.com',
+              'label': 'dev@demistodev.onmicrosoft.com',
+              'type': 'user'}]
+}
+
+ALERT_BY_ID_DATA = {
```

Comment:
please move all of these CONSTS to another folder called test_data. 
The file will be more readable this way